.github/workflows/stale-bot.yaml

```diff
@@ -10,11 +10,15 @@ jobs:
       - uses: actions/stale@v8
         with:
           stale-issue-message: '' # no comment left if string is empty
-          stale-pr-message: '' # no comment left if string is empty
+          stale-pr-message: "This pull request is stale because it's been open for 60 days with no activity. Please update and respond to this comment if you're still interested in working on this. It will be closed after 7 days of inactivity."
+          close-pr-message: "This pull request was closed as stale."
```

Comment:
How about:

"This pull request was closed due to lack of activity. Please reopen if you'd like to continue engaging."

It'd be good to indicate that reopening is an option